### ISSUE TITLE
WebDAV Synchronization

### ISSUE THREAD

**Issue by uGeek** [msoultanidis/quillnote#40]
Synchronization with WebDAV would be possible?

There are many clouds that allow WebDAV such as Nextcloud, ownCloud, FileRun, seafile, Box.com, Yamdex, ... Also, it is very easy to set up a simple and lightweight WebDAV server with docker.

Thank you very much and Great application. Congratulations

**Comment by Zeg31** [comment]
+1

(Nice App, thank you!)

**Comment by KonstantMan** [comment]
Yeah it would be nice to have this option since i already use webdav for calendars and tasks. And i believe it would be easier for you to let another app handle the sync for you.

Also because i saw another issue for https, you could add the option for webdav if someone uses http

**Comment by jivvy** [comment]
WebDAV would be monumental. Nextcloud syncing is good, but Nextcloud can also sync using WebDAV. This is what I'm currently doing with Joplin.
Being able to move from Joplin to a nicer UX like Quill would be very nice :)

**Comment by msoultanidis** [comment]
Next update will feature the ability to sync notes locally, as files. That means you will be able to sync with any provider that supports files, including Nextcloud, with their application.

**Comment by KonstantMan** [comment]
Although it's a nice and versatile option to save everything as a file and then sync it. I currently see 2 problems with it
1 I don't know how secure it is to just save notes as a plain txt file
2 Not all cloud provides like nextcloud or desktop note taking apps have an option to sync notes with a txt file you have in  cloud. The point of webdav is to easily sync what you want with simply just a refresh and not moving txt files around and having to manually sync most of the time.

**Comment by uGeek** [comment]
It is a great idea since it allows you to synchronize with the tool you want and in a very open mode.  

Having the files in plain text is a great.  

Anyway, WebDAV is a very stable and unmodified tool, unlike other synchronization services that constantly modify their API.

It is more stable to add synchronization for Nextcloud - Owncloud by WebDAV than using the APIs of these services. In addition, it allows you to synchronize with many more services such as Box.com, Yandex, ...

**Comment by ltGuillaume** [comment]
I fully agree @uGeek 

If any sync should be implemented, I'd say WebDAV is the one.

**Comment by thoronaug** [comment]
This app is beautiful and very handy. I would love if I could sync my notes with my pCloud space via WebDAV. Thanks!
